File: semana2/Ponto/src/Ponto/Ponto.java
package Ponto;

public class Ponto {
    private float x, y;

    public Ponto(float x, float y) {
        this.x = x;
        this.y = y;
    }
    public Ponto() {
        this(0, 0);
    }

    public float getX() {
        return x;
    }
    public void setX(float x) {
        this.x = x;
    }

    public float getY() {
        return y;
    }
    public void setY(float y) {
        this.y = y;
    }
}

File: semana02/Ponto/src/Ponto.cpp
class Point {
    private:
        float x, y;

    public:
        Point(float x, float y) {
            this->x = x;
            this->y = y;
        }

        float getX() {
            return x;
        }

        float getY() {
            return y;
        }

        void setX(float newX) {
            x = newX;
        }

        void setY(float newY) {
            y = newY;
        }
};
